advisories/server-list-part.php

```diff
@@ -1,13 +1,17 @@
 <p>Version 9.1.3</p>
-<a href="/security/advisory?id=oc-sa-2016-022">Bypass received read-only share permissions using read-write reshare</a><br>
+<a href="/security/advisory?id=oc-sa-2017-001">User enumeration with error messages</a><br>
+<a href="/security/advisory?id=oc-sa-2017-002">Information disclosure in email field dialog at sharing</a><br>
+<a href="/security/advisory?id=oc-sa-2017-003">Flooding logfiles with a 1 Bit BMP File</a><br>
 <br/><p>Version 9.1.2</p>
 <a href="/security/advisory?id=oc-sa-2016-017">SMB User Authentication Bypass</a><br>
 <a href="/security/advisory?id=oc-sa-2016-018">Stored XSS in CardDAV image export</a><br>
 <a href="/security/advisory?id=oc-sa-2016-019">Reflected XSS in Gallery application</a><br>
 <a href="/security/advisory?id=oc-sa-2016-020">Content-Spoofing in "files" app</a><br>
 <a href="/security/advisory?id=oc-sa-2016-021">Content-Spoofing in "dav" app</a><br>
 <br/><p>Version 9.0.7</p>
-<a href="/security/advisory?id=oc-sa-2016-022">Bypass received read-only share permissions using read-write reshare</a><br>
+<a href="/security/advisory?id=oc-sa-2017-001">User enumeration with error messages</a><br>
```

Comment:
should this say 9.0.8 which is the fixed version ?

any reason you removed "bypass received read-only share perms" ?